fdbcli/BulkDumpCommand.actor.cpp

```diff
@@ -0,0 +1,157 @@
+/*
+ * BulkDumpCommand.actor.cpp
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2024 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <cstddef>
+#include <fmt/core.h>
+#include "fdbcli/fdbcli.actor.h"
+#include "fdbclient/BulkDumping.h"
+#include "fdbclient/IClientApi.h"
+#include "fdbclient/ManagementAPI.actor.h"
+#include "flow/Arena.h"
+#include "flow/FastRef.h"
+#include "flow/ThreadHelper.actor.h"
+#include "flow/actorcompiler.h" // This must be the last #include.
+
+namespace fdb_cli {
+
+ACTOR Future<bool> getOngoingBulkDumpJob(Database cx) {
+	state Transaction tr(cx);
+	loop {
+		try {
+			Optional<UID> jobId = wait(existAnyBulkDumpTask(&tr));
+			if (jobId.present()) {
+				fmt::println("Running bulk dumping job: {}", jobId.get().toString());
+				return true;
+			} else {
+				fmt::println("No bulk dumping job is running");
+				return false;
+			}
+		} catch (Error& e) {
+			wait(tr.onError(e));
+		}
+	}
+}
+
+ACTOR Future<Void> getBulkDumpCompleteRanges(Database cx, KeyRange rangeToRead) {
+	try {
+		size_t finishCount = wait(getBulkDumpCompleteTaskCount(cx, rangeToRead));
+		fmt::println("Finished {} tasks", finishCount);
```

Comment:
Would this be more useful if it was "Finished {} tasks of a total {}" ? Is it easy to get the total count of tasks

Reply:
We are not be able to know total tasks at this point since the task is spawned overtime.

Reply:
There will be a separate PR for the progress visibility and speed adjustment to meet the period deadline.

Reply:
Sounds good.